src/Foundation/NSUnit.cs

```diff
@@ -20,107 +20,102 @@ public partial class NSUnit {
 	}
 
 	public partial class NSUnitElectricCurrent {
 		[Obsolete ("Use .ctor(string, NSUnitConverter) or any of the static properties.")]
-		public NSUnitElectricCurrent () { }
+		public NSUnitElectricCurrent () : base (NSObjectFlag.Empty) { }
 	}
 
 	public partial class NSUnitElectricPotentialDifference {
 		[Obsolete ("Use .ctor(string, NSUnitConverter) or any of the static properties.")]
-		public NSUnitElectricPotentialDifference () { }
+		public NSUnitElectricPotentialDifference () : base (NSObjectFlag.Empty) { }
 	}
 
 	public partial class NSUnitElectricResistance {
 		[Obsolete ("Use .ctor(string, NSUnitConverter) or any of the static properties.")]
-		public NSUnitElectricResistance () { }
+		public NSUnitElectricResistance () : base (NSObjectFlag.Empty) { }
 	}
 
 	public partial class NSUnitEnergy {
 		[Obsolete ("Use .ctor(string, NSUnitConverter) or any of the static properties.")]
-		public NSUnitEnergy () { }
+		public NSUnitEnergy () : base (NSObjectFlag.Empty) { }
 	}
 
 	public partial class NSUnitFrequency {
 		[Obsolete ("Use .ctor(string, NSUnitConverter) or any of the static properties.")]
-		public NSUnitFrequency () { }
+		public NSUnitFrequency () : base (NSObjectFlag.Empty) { }
 	}
 
 	public partial class NSUnitFuelEfficiency {
 		[Obsolete ("Use .ctor(string, NSUnitConverter) or any of the static properties.")]
-		public NSUnitFuelEfficiency () { }
+		public NSUnitFuelEfficiency () : base (NSObjectFlag.Empty) { }
 	}
 
 	public partial class NSUnitLength {
 		[Obsolete ("Use .ctor(string, NSUnitConverter) or any of the static properties.")]
-		public NSUnitLength () { }
+		public NSUnitLength () : base (NSObjectFlag.Empty) { }
 	}
 
 	public partial class NSUnitIlluminance {
 		[Obsolete ("Use .ctor(string, NSUnitConverter) or any of the static properties.")]
-		public NSUnitIlluminance () { }
+		public NSUnitIlluminance () : base (NSObjectFlag.Empty) { }
 	}
 
 	public partial class NSUnitMass {
 		[Obsolete ("Use .ctor(string, NSUnitConverter) or any of the static properties.")]
-		public NSUnitMass () { }
+		public NSUnitMass () : base (NSObjectFlag.Empty) { }
 	}
 
 	public partial class NSUnitPower {
 		[Obsolete ("Use .ctor(string, NSUnitConverter) or any of the static properties.")]
-		public NSUnitPower () { }
+		public NSUnitPower () : base (NSObjectFlag.Empty) { }
 	}
 
 	public partial class NSUnitPressure {
 		[Obsolete ("Use .ctor(string, NSUnitConverter) or any of the static properties.")]
-		public NSUnitPressure () { }
+		public NSUnitPressure () : base (NSObjectFlag.Empty) { }
 	}
 
 	public partial class NSUnitSpeed {
 		[Obsolete ("Use .ctor(string, NSUnitConverter) or any of the static properties.")]
-		public NSUnitSpeed () { }
+		public NSUnitSpeed () : base (NSObjectFlag.Empty) { }
 	}
 
 	public partial class NSUnitVolume {
 		[Obsolete ("Use .ctor(string, NSUnitConverter) or any of the static properties.")]
-		public NSUnitVolume () { }
-	}
-
-	public partial class NSDimension {
```

Comment:
yes, see commit message - it was adding a new, obsolete .ctor which is not required when chaining differently.